README.md

```diff
@@ -30,17 +30,22 @@ IF YOU WANT TO COMPILE THE GAME YOURSELF, CONTINUE READING!!!
 
 First, you need to install Haxe and HaxeFlixel. I'm too lazy to write and keep updated with that setup (which is pretty simple). 
 1. [Install Haxe 4.1.5](https://haxe.org/download/version/4.1.5/) (Download 4.1.5 instead of 4.2.0 because 4.2.0 is broken and is not working with gits properly...)
-2. [Install HaxeFlixel](https://haxeflixel.com/documentation/install-haxeflixel/) after downloading Haxe
+2.  You can [Install HaxeFlixel](https://haxeflixel.com/documentation/install-haxeflixel/) after downloading Haxe or you can download separately openfl, lime and hxcpp
 
 Other installations you'd need are the additional libraries, a fully updated list will be in `Project.xml` in the project root. Currently, these are all of the things you need to install:
 ```
 flixel
+openfl 9.0.2
+lime 7.7.0
+hxcpp 4.1.15
+vshaxe
 flixel-addons
 flixel-ui
 hscript
 newgrounds
 ```
 So for each of those type `haxelib install [library]` so shit like `haxelib install newgrounds`
+(Downloading the openfl, lime and hxcpp libraries with each specific version and vshaxe can make you use the Haxe [language server](https://marketplace.visualstudio.com/items?itemName=nadako.vshaxe) in Visual Studio Code, You can't use another type of versions without failing your cache, making it very slow and tedious.)

```

Comment:
This is incoherent. Rephrase it as a suggestion for those specifically using VS Code/VS Codium.